systemtests/tests/py3plug-fd-postgresql/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+#   BAREOS® - Backup Archiving REcovery Open Sourced
+#
+#   Copyright (C) 2021-2023 Bareos GmbH & Co. KG
+#
+#   This program is Free Software; you can redistribute it and/or
+#   modify it under the terms of version three of the GNU Affero General Public
+#   License as published by the Free Software Foundation and included
+#   in the file LICENSE.
+#
+#   This program is distributed in the hope that it will be useful, but
+#   WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#   Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program; if not, write to the Free Software
+#   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+#   02110-1301, USA.
+
+if(TARGET python3-fd)
+  check_pymodule_available(3 pg8000)
+  check_pymodule_available(3 dateutil)
+  include(FindPostgreSQL)
+  if(PYMODULE_3_PG8000_FOUND
+     AND PostgreSQL_FOUND
+     AND PYMODULE_3_DATEUTIL_FOUND
+     AND (${Python3_VERSION_MAJOR} EQUAL 3)
+     AND (${PostgreSQL_VERSION_STRING} VERSION_GREATER "10.0")
```

Comment:
Since this is the old plugin, should we not disable it if the version is >= 15.0 here ?

Reply:
@sebsura the code you're showing is the new postgresql plugin.

we already disable the old one if PG is >= 15 

in `systemtests/tests/py3plug-fd-postgres/CMakeLists.txt`
```
  if(PYMODULE_3_PG8000_FOUND
     AND PostgreSQL_FOUND
     AND PYMODULE_3_DATEUTIL_FOUND
     AND (${Python3_VERSION_MAJOR} EQUAL 3)
     AND (${PostgreSQL_VERSION_STRING} VERSION_LESS "15.0")
  )
```

Reply:
I think its good to leave it on for now.  Until we are ready to drop the plugin, its best to still test it.

Reply:
ok then we are ok.